PeerioTechnologies/peerio-desktop
# improve localisation library

#### Relevant info and issue/PR links

ch story:
https://app.clubhouse.io/peerio/story/13663/improve-localisation-library

icebear branch:
https://github.com/PeerioTechnologies/peerio-icebear/pull/347

After merge, **all new imports of `t` should be from `peerio-icebear`, not `peerio-translator`**, so we should let folks know.

#### Testing instructions

No functionality should be changed.

New runtime warnings may be being generated for missing localization keys -- maybe click around a bit and see if you spot any after merge?

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
